### DISCUSSION TITLE
Bacon and Eggs ?

### DISCUSSION THREAD

**Discussion by Airfiddler** [EdgeTX/edgetx#2803] [General]
I was wondering if 2.9 could have a function we could put on a switch so EDGETX would fry me up some Bacon and eggs, on the days I go to the field really early. I think this would be a great enhancement. EDGETX seems to have everything else covered, so we need to think outside the transmitter.
Seriously..... amazing work development team. Love 2.8.0.

**Comment by pfeerick** [comment]
I'm sure you could do something with a Lua script for that... might need some assistance though... some servos, motors, maybe a robotic arm or two... some raspberry pis, cameras, ... totally doable though ;) 😆  

But seriously... glad you like it... 

🥓 🍳
